monitor_ingest/config.schema.yaml

```diff
@@ -0,0 +1,132 @@
+---
+action_type:
+  description: "Action type:  SetupEntityAction | SetupAddConstants | SetupAddDimesions | SetupAddFunctions "
```

Comment:
If this is only allowed to be specific values of strings, you should use the `enum` classifier: https://github.com/StackStorm/st2/blob/master/contrib/core/actions/http.yaml#L33-L42

Reply:
updated the file to use enum